classes/acrn-sblimage.bbclass

```diff
@@ -0,0 +1,88 @@
+inherit python3native
+DEPENDS += "slimboot-tools-native"
+
+MB_DEPENDENCY    ?= ""
+MB_MC_DEPENDENCY ?= ""
+MB_ACRN_BINARY   ?= "${TMPDIR}/deploy/images/${MACHINE}/acrn.32.out"
+MB_ACRN_CMDLINE  ?= ""
+MB_ACRN_MODULES  ?= "${TOPDIR}/conf/linux.txt;${TMPDIR}/deploy/images/${MACHINE}/${KERNEL_IMAGETYPE}"
+BASE_SBLIMAGE    ?= "sbl_os"
+SBLIMAGE_NAME    ?= "${BASE_SBLIMAGE}"
+PREGENERATED_SIGNING_KEY_SLIMBOOT_KEY_SHA256 ?= "${TOPDIR}/cert/TestSigningPrivateKey.pem"
```

Comment:
I think it'll help if the link to documentation that you've provided in pull request is given here.

Reply:
I will create a md doc file and describe them.